themes/default/layouts/index.html

```diff
@@ -271,29 +267,47 @@ <h6>{{ $item.title }}</h6>
         </div>
     </section>
 
-    <!-- Benefits -->
-    <section class="container mx-auto my-12 lg:my-24 relative">
-        <h2 class="text-center">{{ .Params.benefits.title }}</h2>
-        <div class="benefits-section relative">
-            <div class="shape-background home-benefits-section">
-                <div class="shape-container">
-                    <div class="circle-left"></div>
-                    <div class="circle-right"></div>
+    <section class="container mx-auto my-16 text-center px-3">
+        <div class="card bg-violet-200 p-6 rounded-2xl">
+            <h2 class="rainbow-text">Doing more with less</h2>
+            <p class="text-lg max-w-3xl mx-auto">
+                You can use Pulumi's Automation API to create tooling that help you and your engineers manage <span class="font-bold">10x the amount of resources</span> versus
```

Comment:
here's another messaging I wrote up for this section for consideration:

> Pulumi enables you to do more with less because you can program IaC automation in addition to running manual CLI commands. You can build infrastructure automation directly into your applications or custom workflows by using the Pulumi engine as a library, enabling them to create, update, and destroy infrastructure. This helps you 10x your productivity because the volume of infrastructure operations you can perform is only limited by the code running it.